spinnaker-monitoring/spectator_client.py

```diff
@@ -33,6 +33,7 @@
 
 
 CONFIG_DIR = '/opt/spinnaker-monitoring/config'
+SOURCES_DIR = '/opt/spinnaker-monitoring/sources'
```

Comment:
I think this should default to config/sources and you can override it for docker.
@lwander I forget why we needed to break this out. It had something to do with how he was trying to mount the directories and we couldnt mount a subdir onto a mounted parent dir. I think. @ttomsu could you explain and see if lars has a different solution? I'd like to try to keep these dirs together since I think it would be simpler to manage (but could be wrong).